client/apps/plugin-status/test/health.test.js

```diff
@@ -76,18 +81,40 @@ describe('Health View', () => {
 
 		const sections = wrapper.find('fieldset.form-fieldset')
 
-		expect(sections).toHaveLength(3)
+		expect(sections).toHaveLength(4)
 
 		const wooCommerceSection = sections.at(0)
 		const jetpackSection = sections.at(1)
-		const wcsSection = sections.at(2)
+		const taxesSection = sections.at(2)
+		const wcsSection = sections.at(3)
 
 		expect(wooCommerceSection.text()).toContain('WooCommerce 5.0.0 is configured correctly')
 		expect(jetpackSection.text()).toContain('Jetpack 9.4 is connected and working correctly')
+		expect(taxesSection.text()).toContain('Automatic taxes functionality is enabled on your store')
+		expect(taxesSection.text()).toContain('Go to the Tax settings')
 		expect(wcsSection.text()).toContain('Service data is up-to-date')
 		expect(wcsSection.text()).toContain('Last updated 3 minutes ago')
 	})
 
+	it('should display not display the settings link when not needed', () => {
+		const wrapper = mount(
+			<Wrapper healthStoreOverrides={{
+				automatic_taxes: {
+					state: 'warning',
+					message: 'TaxJar extension detected. Automatic taxes functionality is disabled',
+					show_settings_link: false,
+				}
+			}}>
+				<HealthView/>
+			</Wrapper>
+		)
+
+		const wcsSection = wrapper.find('fieldset.form-fieldset').at(2)
```

Comment:
We're actually not targeting `wcsSection` here (which is the "WooCommerce Shipping & Tax Data" section in the other tests).

```suggestion
		const taxSection = wrapper.find('fieldset.form-fieldset').at(2)
```